src/components/Header/index.tsx

```diff
@@ -0,0 +1,47 @@
+'use client';
+
+import EmailButton from '@components/Header/EmailButton';
+import HamburgerBotton from '@components/Header/HamburgerBotton';
+import LogoButton from '@components/Header/LogoButton';
+import NotificationButton from '@components/Header/NotificationButton';
+import useRWD from '@hook/useRWD';
+import { useAuthContext } from 'src/store/AuthProvider/AuthProvider';
+
+export type UserProfileButtonProps = {
+  onSignOut?: () => void;
+};
+
+const Header = () => {
+  const { isLoggedin, logout } = useAuthContext();
+  const device = useRWD();
+
+  const handleSignOut: UserProfileButtonProps['onSignOut'] = () => {
+    logout();
+  };
+
+  return (
+    <>
+      <header className="flex w-full justify-center bg-[#e8e4de] px-3">
+        <div className="flex h-[112px] w-full max-w-[1296px] items-center justify-between bg-[#e8e4de]  ">
+          <LogoButton />
+          <div className="flex">
```

Comment:
建議：應由元件外部去控制每個元件的寬度，tailwindCSS有另外提供space的語法，可以控制非flex元素的tag，詳情至EmailButton的Review
```suggestion
          <div className="flex gap-4">
```